### DISCUSSION TITLE
Installing Windows Server 2003 on UTM Crashed

### DISCUSSION THREAD

**Discussion by allamiro** [utmapp/UTM#5187] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
Replace this text with a clear and concise description of what the issue is.

**Configuration**  
* UTM Version: Version 4.1.6 (75)
* macOS Version: Ventura 13.3.1 (22E261)
* Mac Chip (Intel, M1, ...):  M1

**Crash log**  
see screenshot

**Debug log**  
attached

**Upload VM**  
uploaded 

![Screenshot 2023-04-09 at 6 28 14 PM]
[config.plist.zip](https://github.com/utmapp/UTM/files/11185867/config.plist.zip)
(https://user-images.githubusercontent.com/279790/230784602-58d6edf2-0083-4172-ad38-a5a9fcec5022.png)

**Comment by osy** [comment, marked as answer]
Older versions of windows require “pc” machine not q35

**Reply by Arfench** [reply]
on version 3.2.4 when selecting pc, it freezes on the first pre-installation launch from cd
oc: ios 12.5.1
